v1/linkml-schemas/person.yaml

```diff
@@ -53,4 +54,8 @@ slots:
   drivingPermit:
     range: DrivingPermitDetailsClass
     multivalued: true
-    inlined_as_list: true
\ No newline at end of file
+    inlined_as_list: true
```

Comment:
We should check why we're doing this for drivingPermit but not for passport? I think they should be consistent.

Reply:
Yeah I will take a look, I think it was probably just missing from the beginning, but no real reason why.  If I can't find any evidence as to why it needs to be different I will update.